float64 leeway
int8 navcase
float64 err
float64 rudderPCont
float64 rudderICont
float64 rudderDCont
